apps/docs-snippets2/src/utilities/unit-conversion.ts

```diff
@@ -0,0 +1,113 @@
+// #region full
+import { BN, DECIMAL_GWEI, DECIMAL_KWEI, bn, Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../env';
+import { EchoValuesFactory } from '../typegend/contracts/EchoValuesFactory';
+
+// #region instantiation-1
+const myBigNumberOne = '100000000';
+
+const resultOne = new BN('100000000').toString();
+console.assert(resultOne === myBigNumberOne, 'BN is not equal to expected');
```

Comment:
```suggestion
const resultOne = new BN('100000000').toString();

console.assert(resultOne === myBigNumberOne, 'BN is not equal to expected');
```

Maybe some whitespace between these will help readability. Same goes for all on this file.